10/1 Cloud release notes (#116)
astronomer/cloud-docs: docs/release-notes.md

**Latest Runtime Version**: 3.0.3 ([Release notes](runtime-release-notes))
**Latest CLI Version**: 1.0.0 ([Release notes](cli-release-notes))

## October 1, 2021

### Minor Improvements

> - In the Astronomer UI, the **Open Airflow** button is now disabled until a Deployment is live and healthy 

Reviewer: ```suggestion
- In the Astronomer UI, the **Open Airflow** button is now disabled until the Airflow UI of the Deployment is available.
```

> - Workspace Admins can now both edit user permissions and remove users within a given Workspace

Reviewer: ```suggestion
- Workspace Admins can now edit user permissions and remove users within a given Workspace.
```
